src/nri/sampler.go

```diff
@@ -69,6 +69,17 @@ func (cs *ContainerSampler) SampleAll(ctx context.Context, i *integration.Integr
 		return err
 	}
 
+	//TODO Add ff to get this info
```

Comment:
I would add a config option to disable the collection of these metrics.

Reply:
Forgot to address that , thanks! I can add a config file to disable this metric collection.